### ISSUE TITLE
Minor updates to Download section

### ISSUE THREAD

**Issue by jdaudier** [marionettejs/marionettejs.com#267]
Fix this...
![2015-02-18_1419](https://cloud.githubusercontent.com/assets/1666031/6254795/2f4a4a8e-b779-11e4-8a38-9d81dfada6e9.png)

Maybe clicking the "Marionette" text should return you to homepage instead of scroll to top since this page is shorter than homepage?

![2015-02-18_1418](https://cloud.githubusercontent.com/assets/1666031/6254794/2f486c78-b779-11e4-9ed8-2ca473ded8d6.png)

**Comment by peterblazejewicz** [comment]
@jdaudier We know that the second one is an actual UX bug. Let's fix this

**Comment by jdaudier** [comment]
@peterblazejewicz you mean you agree it should go to the homepage, right?

**Comment by peterblazejewicz** [comment]
@jdaudier Yes, we should ditch now this scroll to top pattern attached to logo/brand - as there will be multiple pages (a.k.a website with common parts)

**Comment by peterblazejewicz** [comment]
x-ref: #270 I made an issue for this
